packages/lit-dev-tools-cjs/src/api-docs/transformer.ts

```diff
@@ -568,6 +573,85 @@ export class ApiDocsTransformer {
     return a.name.localeCompare(b.name);
   };
 
+  /**
+   * Temporarily turn more modern formats of TypeDoc into a legacy format that
+   * our HTML template understands.
+   */
+  private replaceCommentNodesWithLegacyCommentFields(
+    node: DeclarationReflection
+  ) {
+    if (node.comment) {
+      this.replaceCommentNodeForLegacyFormat(node.comment);
+    }
+    if (node.signatures) {
+      for (const sig of node.signatures) {
+        if (sig.parameters) {
+          if (sig.comment) {
+            this.replaceCommentNodeForLegacyFormat(sig.comment);
+          }
+          for (const param of sig.parameters) {
+            if (param.comment) {
+              this.replaceCommentNodeForLegacyFormat(param.comment);
+            }
+          }
+        }
+      }
+    }
+    if (node.getSignature?.comment) {
+      this.replaceCommentNodeForLegacyFormat(node.getSignature?.comment);
+    }
+    if (node.type?.type === 'reflection' && node.type.declaration) {
+      const declaration = node.type.declaration;
+      if (declaration.comment) {
+        this.replaceCommentNodeForLegacyFormat(declaration.comment);
+      }
+      for (const sig of declaration.signatures ?? []) {
+        if (sig.comment) {
+          this.replaceCommentNodeForLegacyFormat(sig.comment);
+        }
+        for (const param of sig.parameters ?? []) {
+          if (param.comment) {
+            this.replaceCommentNodeForLegacyFormat(param.comment);
+          }
+        }
+      }
+    }
+  }
+
+  private replaceCommentNodeForLegacyFormat(commentNode: MigrationComment) {
```

Comment:
This name might also be better as something along the lines of. replaceCommentNodePartsWithMarkdown?